docs/guide/adapters/emberznet.md

```diff
@@ -168,6 +168,23 @@ serial:
 
 :::
 
+::: details Easyiot ZB-GW04 (v1.1, v1.2)
+
+Chip: [efr32mg21a020f768im32](https://www.silabs.com/wireless/zigbee/efr32mg21-series-2-socs/device.efr32mg21a020f768im32?tab=specs)
```

Comment:
If this chip did not change, I guess it is still using low resource hardware and should stay under not recommended?